src/FakeItEasy/Configuration/BuildableCallRule.cs

```diff
@@ -11,7 +11,7 @@ namespace FakeItEasy.Configuration
     internal abstract class BuildableCallRule
         : IFakeObjectCallRule
     {
-        private readonly List<WherePredicate> wherePredicates;
+        private readonly SynchronizedCollection<WherePredicate> wherePredicates;

```

Comment:
Is that a reason to _not_ make the change? If it's not going to hurt anything, it'd be nice to cover as much as possible.